migration/migrator/migrations/master/20240717141512_self_registration.py

```diff
@@ -0,0 +1,34 @@
+"""Migration for a given Submitty course database."""
+def up(config, database):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    # Self Registration Types: 0 == None, 1 == All users auto join when registering, 2 == Users request to join when registering (added later)
```

Comment:
Would it make sense to swap 1 and 2 here?  That way, increasing values correspond to increasing permissions, and it's thus possible to do greater-than and less-than checks.